ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -42,14 +42,15 @@ const TYPES = [
 ];
 
 angular.module(MODULE_NAME, [angularAnimate, uibModal, brUtils])
-    .directive('catalogSaver', ['$rootScope', '$uibModal', '$injector', 'composerOverrides', 'blueprintService', saveToCatalogModalDirective])
+    .directive('catalogSaver', ['$rootScope', '$uibModal', '$injector', '$filter', 'composerOverrides', 'blueprintService', saveToCatalogModalDirective])
     .directive('catalogVersion', ['$parse', catalogVersionDirective])
+    .directive('blueprintNameOrSymbolicNameAndBundleIdRequired', blueprintNameOrSymbolicNameAndBundleIdRequiredDirective)
```

Comment:
Wow, that a very loooooong name. Could we come up with something shorter like `blueprintNameValidator`? The name doesn't need to say what it does exactly

Reply:
okay, compromise, changed to `composerBlueprintNameValidator`

Reply:
Sounds good 👍